src/js/control-bar/fullscreen-toggle.js

```diff
@@ -5,24 +8,26 @@
  * @class
  * @extends vjs.Button
  */
-vjs.FullscreenToggle = vjs.Button.extend({
+var FullscreenToggle = Button.extend({
   /**
    * @constructor
    * @memberof vjs.FullscreenToggle
    * @instance
    */
   init: function(player, options){
-    vjs.Button.call(this, player, options);
+    Button.call(this, player, options);
   }
 });
 
-vjs.FullscreenToggle.prototype.buttonText = 'Fullscreen';
+Component.registerComponent('FullscreenToggle', FullscreenToggle);
+
+FullscreenToggle.prototype.buttonText = 'Fullscreen';
 
-vjs.FullscreenToggle.prototype.buildCSSClass = function(){
-  return 'vjs-fullscreen-control ' + vjs.Button.prototype.buildCSSClass.call(this);
+FullscreenToggle.prototype.buildCSSClass = function(){
+  return 'vjs-fullscreen-control ' + Button.prototype.buildCSSClass.call(this);
```

Comment:
We can use the template strings here:
```js
`vjs-fullscreen-control ${Button.prototype.buildCSSClass.call(this)}`
```
In *this* particular case it isn't *that* helpful since it just gets compiled down to what we have, but thought I'd point it out.